src/components/BurgerMenu/BurgerMenu.module.scss

```diff
@@ -0,0 +1,86 @@
+@use '~/styles/mixins' as *;
+
+.link {
+	padding: 7px 0;
+	@include useScreen('tablet-sm') {
+		padding: 15px 0;
+	}
+}
+
+.column {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	padding: 40px 58px 0 16px;
+	z-index: 103;
+	gap: 40px;
```

Comment:
think need overflow: auto here ('.column')
or in Navigation (as you like)

Reply:
i think that it reasonable if the navigation will be extended, but now content in burger menu will be displayed without scrolling on any screens

Reply:
ok, try open code on your mobile and rotate 90 deg. After it try to get  button "get help".